server/src/test/models/PushSubscriptionModel-test.ts

```diff
@@ -0,0 +1,106 @@
+import test from 'ava';
+
+import {PushSubscriptionModel} from '../../models/PushSubscriptionModel';
+
+const USER_LOGIN = 'user-login';
+const PUSH_SUBSCRIPTION = {
+  endpoint: 'https://project-health.appspot.com/push-service/123',
+  keys: {
+    'auth': 'auth-123',
+    'p256dh': 'p256dh-123',
+  },
+  expirationTime: null,
+};
+const PUSH_SUBSCRIPTION_2 = {
+  endpoint: 'https://project-health.appspot.com/push-service/456',
+  keys: {
+    'auth': 'auth-456',
+    'p256dh': 'p256dh-456',
+  },
+  expirationTime: null,
+};
+const CONTENT_ENCODINGS = ['aes128gcm', 'aesgcm'];
+
+test(`constructor`, (t) => {
```

Comment:
Can we switch these to '' instead of \`\` please? This would be consistent with our other test files.